### DISCUSSION TITLE
My Flood is not listen to my host ip

### DISCUSSION THREAD

**Discussion by levixdc** [jesec/flood#130] [Q&A] [answered]
hey!

I modified the config.js file to listen to my host ip address.

But when i'll start the process still listen to 127.0.0.1 address.

When i use the --host parameter it works fine
How can i modify correctly the host ip address when i use systemd?

Thanks

**Comment by jesec** [comment, marked as answer]
It is recommended to use CLI arguments. Static configuration is discouraged. It simply won't work unless you compiled Flood from sources. 

See [Install Flood as a service (systemd)](https://github.com/jesec/flood/wiki/Install-Flood-as-a-service-%28systemd%29). 

To add arguments, for example `--host 0.0.0.0`, simply append to `ExecStart`:

`ExecStart=/usr/bin/flood --host 0.0.0.0`

instead of

`ExecStart=/usr/bin/flood`

If you use `npm run start`, append arguments after `--`. See `README`.

**Reply by levixdc** [reply]
Thank you! That was i needed! 👍

**Reply by levixdc** [reply]
Another question, Why are the  main counters (top left) not counting properly? Shows way more than it should and constantly growing

**Reply by jesec** [reply]
Do you mean data downloaded/uploaded in a session? It should count properly. What's your torrent client? How does it differ from the "proper" count?

**Reply by levixdc** [reply]
[Here is the current upload speed, this is good,i checked in my VM and my monitoring system](https://imgur.com/y52aJe6)

[Here is the main counter at the same time, way more](https://imgur.com/X0hQBf7)

I use rtorrent 0.9.6

**Reply by jesec** [reply]
Main counter updates faster than torrent list. As a result, sometimes the overall rate may not equal the sum of rates of all torrents. Flood uses `throttle.global_down.rate` rTorrent command to get current overall rate, and relevant codes are pretty stable.

What's your language? Do the rates display correctly if you use English?

**Reply by levixdc** [reply]
Before you wrote me,the counters started  working well now, but i can't add torrents to download.

My language is set to English

**Reply by jesec** [reply]
See https://github.com/jesec/flood/discussions/86.

**Reply by levixdc** [reply]
Thank you for all the help! Keep up the good work! 👍 👍
